src/components/SearchBar.js

```diff
@@ -0,0 +1,27 @@
+import { useState } from "react";
+import RecipeGallery from "./RecipeGallery";
+
+function SearchBar() {
+
+    const [userInput, setUserInput] = useState("");
+
+
+    const handleInput = (event) => {
+        setUserInput(event.target.value)
+    }
+
+    console.log(userInput)
+    return(
+        <>
+            <form id="searchContainer">
+                <input type="text" value={userInput} placeholder="Search Recipes" onChange={handleInput} />
+            </form> 
+            
+            
+            <RecipeGallery userInput={userInput}/>
+        </>
+    )
+}
```

Comment:
Overall, I think your project is so great, Terri!! I think once you're able to address the small instance of error handling, it'll be *chef's kiss* and everything else (likes, firebase) will just be an awesome bonus!